File: Beautiful-Binary-String.cpp
int beautifulBinaryString(string b)
{
    int result = 0;
    
    for (int i = 0; i < b.size(); ++i)
    {
        if (b.substr(i, 3) == "010")
        {
            result++;
            i += 2;#include <bits/stdc++.h>

using namespace std;

string ltrim(const string &);
string rtrim(const string &);

/*
 * Complete the 'beautifulBinaryString' function below.
 *
 * The function is expected to return an INTEGER.
 * The function accepts STRING b as parameter.
 */

int beautifulBinaryString(string b)
{
    int result = 0;
    
    for (int i = 0; i < b.size(); ++i)
    {
        if (b.substr(i, 3) == "010")
        {
            result++;
            i += 2;
        }
    }    

    return result;
}

File: Beautiful-Binary-String.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Result
{

    /*
     * Complete the 'beautifulBinaryString' function below.
     *
     * The function is expected to return an INTEGER.
     * The function accepts STRING b as parameter.
     */

    public static int beautifulBinaryString(string b)
    {
        int result = 0;
        
        for (int i = 0; i <= b.Length - 3; ++i)
        {
            if (b.Substring(i, 3) == "010")
            {
                result++;
                i += 2;
            }
        }    

        return result;
    }
}


class Solution
{
    public static void Main(string[] args)
    {
        TextWriter textWriter = new StreamWriter(@System.Environment.GetEnvironmentVariable("OUTPUT_PATH"), true);

        int n = Convert.ToInt32(Console.ReadLine().Trim());

        string b = Console.ReadLine();

        int result = Result.beautifulBinaryString(b);

        textWriter.WriteLine(result);

        textWriter.Flush();
        textWriter.Close();
    }
}
